test/src/sdk-harness/test_projects/tx_fields/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "tx_predicate"
+
+[dependencies]
+std = { path = "../../../../../sway-lib-std" }
```

Comment:
```suggestion
std = { path = "../../../../../sway-lib-std" }

```

Reply:
weird, locally this file already has a newline at the end.